src/engine/currencyEngine.js

```diff
@@ -2,6 +2,7 @@
 
 import { bns } from 'biggystring'
 import { type Disklet } from 'disklet'
+import { InsufficientFundsError } from 'edge-core-js'
```

Comment:
No, you can only use the `/types` library in these repos. Anything else will completely ruin the build. Put this back into `edge-core-js/types` ASAP!!!!